.github/CODEOWNERS

```diff
@@ -10,37 +10,79 @@
+/inputFiles/materialPointMethod                       @homel1 @cmcrook5 
+/inputFiles/meshGeneration                            @rrsettgast @cssherman
+/inputFiles/multiphaseFlowFractures                   @CusiniM @paveltomin @jhuang2601
+/inputFiles/multipleMeshBodies                        @rrsettgast @wrtobin
+/inputFiles/phaseField                                @CusiniM @frankfeifan 
+/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin @jhuang2601 @ryar9534 
+/inputFiles/poromechanicsFractures                    @CusiniM @castelletto1 @paveltomin @jhuang2601
+/inputFiles/proppant                                  @rrsettgast @jhuang2601 
+/inputFiles/relpermDriver                             @dkachuma @paveltomin @jafranc
+/inputFiles/simplePDE                                 @rrsettgast @castelletto1 
+/inputFiles/singlePhaseFlow                           @CusiniM @castelletto1 @paveltomin
+/inputFiles/singlePhaseFlowFractures                  @CusiniM @castelletto1 @paveltomin @jhuang2601 
+/inputFiles/singlePhaseWell                           @CusiniM @paveltomin @tjb-ltk
+/inputFiles/solidMechanics                            @CusiniM @castelletto1 @jhuang2601 
+/inputFiles/surfaceGeneration                         @CusiniM @castelletto1 @rrsettgast 
+/inputFiles/thermalMultiphaseFlow                     @paveltomin @dkachuma
+/inputFiles/thermalSinglePhaseFlowFractures           @CusiniM @castelletto1 @paveltomin @jhuang2601 @frankfeifan
+/inputFiles/thermoPoromechanics                       @paveltomin @jhuang2601 @castelletto1 @frankfeifan
+/inputFiles/thermoPoromechanicsFractures              @CusiniM @castelletto1 @paveltomin @jhuang2601 
+/inputFiles/triaxialDriver                            @rrsettgast @jhuang2601
+/inputFiles/wavePropagation                           @acitrain @sframba
+/inputFiles/wellbore                                  @rrsettgast @jhuang2601
+/inputFiles/wellboreECP                               @rrsettgast 
+/src/coreComponents/LvArray                           @rrsettgast @CusiniM @wrtobin @corbett5 
 /src/coreComponents/codingUtilities                   @rrsettgast @untereiner @corbett5 @wrtobin
 /src/coreComponents/common                            @rrsettgast @MelReyCG @corbett5 @wrtobin
-/src/coreComponents/constitutive                      @rrsettgast @CusiniM @dkachuma @paveltomin @joshua-white
+/src/coreComponents/constitutive                      @rrsettgast @CusiniM @dkachuma @paveltomin @castelletto1
+/src/coreComponents/constitutive/capillaryPressure    @CusiniM @dkachuma @paveltomin
+/src/coreComponents/constitutive/contact              @rrsettgast @CusiniM @matteofrigo5 @Guotong-Ren
+/src/coreComponents/constitutive/diffusion            @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/dispersion           @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/docs                  
```

Comment:
```suggestion
/src/coreComponents/constitutive/docs                  @jhuang2601 
```

Reply:
I would leave this blank so that it's excluded from the codeowners lists.